test/e2e/specs/site-editor/font-library.spec.js

```diff
@@ -186,7 +186,8 @@ test.describe( 'Font Library', () => {
 			await page
 				.getByRole( 'button', { name: 'Headings', exact: true } )
 				.click();
-			await page.getByLabel( 'Font' ).selectOption( 'Exo 2' );
+			await page.getByRole( 'button', { name: 'Font' } ).click();
```

Comment:
```suggestion
			await page.getByRole( 'combobox', { name: 'Font' } ).click();
```

Looking at the rendered HTML, it looks like this should be `role="combobox"`?

Reply:
Thanks for taking a look. I'm looking at how to fix the test locally, but I'm having some difficulties running the tests properly in my local environment.

Reply:
@jeryj fixed it here: https://github.com/WordPress/gutenberg/pull/67118/commits/a10e2e9bb970b73e946bb9155d5b74dbd994ab2a